tests/integ/test_telemetry.py

```diff
@@ -607,6 +607,7 @@ def test_execute_queries_api_calls(session, sql_simplifier_enabled):
             "query_plan_height": query_plan_height,
             "query_plan_num_duplicate_nodes": 0,
```

Comment:
you might want to remove this

Reply:
oh, why is that? we are still sending this. So the query_plan_num_duplicate_nodes is the number of cte nodes. the query_plan_duplicated_node_complexity_distribution is the actual distribution of repeated nodes. so it is different, if we sum all of them together, we will get the actual number of repeated nodes, not the cte nodes.

Reply:
yeah I missed that